### PR TITLE
ArSaFirstTest

### PR DESCRIPTION
**Pull request by ArturSabanadze**
3 Tests
1.Check if Title of the Website is correct.
2.Check if the form "Text Input" is working, try to input some text, check the input text
3.Check "my-disabled" element for the following functions: is disabled, is visible and the placeholder text is correct.
4.Check the color of "my-colors" element

### CODE REVIEW

File: src/test/java/school/redrover/ArSaFirstTest.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import school.redrover.runner.BaseTest;
+
+public class ArSaFirstTest extends BaseTest {
+
+    //Check if Title of the Website is correct..
+    @Test
+    public void titleCheck()
```
Comment (EkaterinaSokolova-V) [inline src/test/java/school/redrover/ArSaFirstTest.java:13]: Please add Test prefix to the test title, should be testTitleCheck.

**Review by Elena-Shagdurova — APPROVED** [review]

File: src/test/java/school/redrover/ArSaFirstTest.java (hunk 1 above)
Comment (EkaterinaSokolova-V) [reply src/test/java/school/redrover/ArSaFirstTest.java:13]: Please check other test titles as well.

Comment (ArturSabanadze) [reply src/test/java/school/redrover/ArSaFirstTest.java:13]: Thanks :)

**Review by agadjan — COMMENTED** [review]
Good

**Review by EkaterinaSokolova-V — APPROVED** [review]

**Review by LiudmilaLobanova — APPROVED** [review]

**Review by agadjan — APPROVED** [review]

**Pull request merged**